base_app/src/public/index.html.txt

```diff
@@ -1,26 +0,0 @@
-<!doctype html>
```

Comment:
Are you sure we can delete this file?

Reply:
I believe this file is left over from when I was trying to figure out everything with `baseHref`. I can't find any references to it within the simpleui project so I think it's okay to delete